queries.sql

```diff
@@ -6,3 +6,32 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+UPDATE animals SET species = 'unspecified';
+UPDATE animals SET species ='';
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species ='';UPDATE animals SET species = 'pokemon' WHERE species ='';
+
```

Comment:
The second transaction as well should have a `begin` statement, a `commit` statement and a `select`statement  to verify the chanes persist, as shown in the screenshot below.
![image](https://user-images.githubusercontent.com/81932478/198305905-3205b38e-4432-40da-b541-6744b52a0b7d.png)